tests/rule_based_profiler/conftest.py

```diff
@@ -29,7 +32,8 @@
 def column_Age_structured_type_domain():
     return Domain(
         domain_kwargs={"column": "Age", "batch_id": "1234567890"},
-        domain_type=StructuredDomainTypes.COLUMN,
+        # TODO: <Alex>ALEX -- Need to discuss the contents of Domain in RuleBasedProfiler</Alex>
```

Comment:
What do you mean by "contents of Domain"?

Reply:
I meant "what are all the properties that we need in the `Domain` class?  But I deleted these TODO comment (this was addressed during the meeting).  Thanks!